README.md

```diff
@@ -55,6 +55,7 @@ Other important settings are:
 * `CONFIG_CLEAN_TEMP_DIR`: Delete temporay directory before starting, by default: `false`
 * `CONFIG_BROKER`: by default: `amqp://localhost:5672`
 * `CONFIG_ROOT_REPOSITORIES_DIR`: .siva file storage. If no HDFS connection url is provided, this will be a path in local filesystem. Otherwise, it will be an HDFS directory, by default: `/tmp/root-repositories`
+* `CONFIG_ROOT_REPOSITORIES_TEMP_DIR`: where `siva.copy` files are temporary placed. If no HDFS connection url is provided, this will be a local path. If not, it will be an HDFS folder, by default: `/tmp/root-repositories-dot-copy`
```

Comment:
This is only used with HDFS storage if I'm not mistaken. LocalCopier does not have a tempdir parameter.